testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1109,3 +1109,8 @@
 Then(/^I should see the text "(.*?)" in the (Operating System|Architecture|Channel Label) field/) do |text, field|
   page.has_field?(text, with: field)
 end
+
+Then(/^I should see the current time as (Start Time|End Time)/) do |timestamp|
+  now = Time.now
+  page.has_field?(now.strftime('%k:%M').to_s, with: timestamp)
```

Comment:
I'd say the .to_s is redundant.
https://docs.python.org/3/library/datetime.html

Reply:
Jup seems to be redundant; removed it :)